.gitignore

```diff
@@ -24,5 +24,6 @@
 
 /config/database.yml
 /config/secrets.yml
+/config/wifi_density.yml
```

Comment:
Does this need to be a secret? Or can we commit it to the repo?

Reply:
It's not a secret, but it may vary from environment to environment as we test new configurations based on uncertain wifi data "high" values.